README.md

```diff
@@ -29,6 +29,18 @@ Options:
   Report bugs at github.com/jennyEckstein/sqslite/issues
 ```
 
+To start the SQSLite server:
+```
+$ sqslite --port=3001
+```
+
+As a prerequisite to interacting with SQSLite through the command line, first install [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html). 
+
+Then, to create a new queue targeting the endpoint the recently created SQSLite server, enter the following command:
```

Comment:
In order to create a queue, run the command shown below. Make sure to provide the same endpoint url where sqslite server is running.